### ISSUE TITLE
Sometimes panel is raised when cursor approaches edge without reaching edge

### ISSUE THREAD

**Issue by AndydeCleyre** [psifidotos/nowdock-panel#61]
This may be a plasma panel issue rather than a nowdock-panel issue.

When moving the cursor *nearly* to the edge, without actually making contact with the edge, *sometimes* this triggers the panel to show (but not always). I would expect the panel to show only when I actually make contact with the edge. The current behavior causes problems when trying to access parts of an application window at that edge.

`0.5.0`

**Comment by psifidotos** [comment]
> When moving the cursor nearly to the edge, without actually making contact with the edge, sometimes this triggers the panel to show (but not always). I would expect the panel to show only when I actually make contact with the edge. 

you are right... this is happening because a hidden window is triggering the dock to show, and I can not make this window thinner... This could be easily solved if we change architecture for 0.6
